raiden/api/python.py

```diff
@@ -452,6 +467,11 @@ def set_total_channel_deposit(
         if channel_state is None:
             raise InvalidAddress('No channel with partner_address for the given token')
 
+        if self.raiden.config['environment_type'] == Environment.PRODUCTION:
+            per_token_network_deposit_limit = RED_EYES_PER_TOKEN_NETWORK_LIMIT
```

Comment:
Wasn't this going to be provided by the smart contract itself?

Reply:
After the latest change in the contracts, limits are no longer hard coded there but rather, they're specified once the contract is instantiated on-chain. For the existing Red eyes contracts, the limits are hard coded and we need to work with them, so that's what i am doing here. 
I am guessing that what you expect is for a query to fetch the limits for the Red eyes contracts and then just use those right? If that's the case, i thought about it and opted for defining the limits in Red eyes since they're not going to change in the contracts anyway so i would shave-off the additional query. Let me know if you think this must be changed.

Reply:
> For the existing Red eyes contracts, the limits are hard coded and we need to work with them, so that's what i am doing here.

So, maintaining support for old contracts versions, but we are not maintaining compatibility across contract networks? why don't we just drop support for the old red eyes altogether?

> Let me know if you think this must be changed.

Not really, a comment would be helpful though. An alternative is to use inheritance and have that optimization hidden in a `RedEyes` subclass.

Reply:
We can't drop support for Red Eyes at this point when we have no other mainnet release ready.

Reply:
> We can't drop support for Red Eyes at this point when we have no other mainnet release ready.

That's okay, but IMO we should not deploy the new smart contracts in the mainnet without the message fix, since the networks will be disjoint.